ci/test/pipeline.template.yml

```diff
@@ -277,9 +277,10 @@ steps:
       queue: linux-x86_64
 
   - id: sqllogictest-fast
-    label: Fast SQL logic tests
+    label: Fast SQL logic tests %n
     depends_on: build-x86_64
     timeout_in_minutes: 30
+    parallelism: 3
```

Comment:
@philip-stoev Is this ok? With `parallelism: 2` one side got unlucky and still took 19 min.

Reply:
Yes more paralellism is fine with me. Ec2 now has per minute billing so the overhead from having more machines involved should be acceptable